buildchain/buildchain/constants.py

```diff
@@ -103,3 +105,13 @@ def git_ref() -> Optional[str]:
 GIT_REF = git_ref()
 
 # }}}
+
+# Only keep directories (except `vendor`) and top-level Go source files.
+STORAGE_OPERATOR_FMT_ARGS : FrozenSet[str] = frozenset([
+    path.name for path in STORAGE_OPERATOR_ROOT.glob('*')
+    if (path.is_dir() and path.name != 'vendor') or path.suffix == '.go'
```

Comment:
There should be no `vendor` directory. If there is one, that's a 'bug'.

Reply:
Ho, you're right!!
The one I had is a vestigial directory of when I was using an old `operator-sdk`.

I guess we could also remove the entry from the `.gitignore` then?

Reply:
Done